src/current/_includes/v24.1/essential-alerts.md

```diff
@@ -0,0 +1,533 @@
+
+### Frequent changefeed restarts
+
+Changefeeds automatically restart in case of transient errors. However "too many" restarts (outside of a routine maintenance procedure) may be due to a systemic condition and should be investigated.
+
+**Metric**
+<br>[`changefeed.error_retries`]({% link {{ page.version.version }}/essential-metrics-{{ include.deployment }}.md %}#changefeed-error-retries)
+
+**Rule**
+<br>WARNING:  If `changefeed.error_retries` is greater than `50` for more than `15 minutes`
+
+**Action**
+
+- Follow the action for a [changefeed failure](#changefeed-failure).
+
+### Changefeed falling behind
+
+Changefeed has fallen behind. This is determined by the end-to-end lag between a committed change and that change applied at the destination. This can be due to cluster capacity or changefeed sink availability.
+
+**Metric**
+<br>[`changefeed.commit_latency`]({% link {{ page.version.version }}/essential-metrics-{{ include.deployment }}.md %}#changefeed-commit-latency)
+
+**Rule**
+<br>WARNING:  `changefeed.commit_latency` is greater than `10 minutes`
+<br>CRITICAL:  `changefeed.commit_latency` is greater than `15 minutes`
+
+**Action**
+
+1. In the DB Console, navigate to **Metrics**, [**Changefeeds** dashboard]({% link {{ page.version.version }}/ui-cdc-dashboard.md %}) for the cluster and check the maximum values on the [**Commit Latency** graph]({% link {{ page.version.version }}/ui-cdc-dashboard.md %}#commit-latency). Alternatively, individual changefeed latency can be verified by using the following SQL query:
+
+    {% include_cached copy-clipboard.html %}
+    ```sql
+    select job_id, status,((high_water_timestamp/1000000000)::int::timestamp)-now() as "changefeed latency",created, left(description,60),high_water_timestamp from crdb_internal.jobs where job_type = 'CHANGEFEED' and status in ('running', 'paused','pause-requested') order by created desc;
+    ```
+
+2. Copy the `job_id` for the changefeed job with highest `changefeed latency` and pause the job:
+
+    {% include_cached copy-clipboard.html %}
+    ```sql
+    PAUSE JOB 681491311976841286;
+    ```
+
+3. Check the status of the pause request by running the query from step 1. If the job status is `pause-requested`, check again in a few minutes.
+
+4. After the job is `paused`, resume the job.
+
+    {% include_cached copy-clipboard.html %}
+    ```sql
+    RESUME JOB 681491311976841286;
+    ```
+
+5. If the changefeed latency does not progress after the above steps due to lack of cluster resources or availability of the changefeed sink, [contact Support](https://support.cockroachlabs.com).
+
+### Changefeed has been paused a long time 
+
+Changefeed jobs should not be paused for a long time because the protected timestamp prevents garbage collections. As a hedge against an operational error, this alert guard against an inadvertently "forgotten" pause. 

```

Comment:
```suggestion
Changefeed jobs should not be paused for a long time because the protected timestamp prevents garbage collection. To protect from an operational error, this alert guards against an inadvertently forgotten pause. 

```

I think you can drop the quotes here. Optional nit: I think "protect" is quicker to parse here and possibly easier for localization efforts.